phpBB/includes/compatibility_globals.php

```diff
@@ -44,7 +44,6 @@
 // Grab global variables, re-cache if necessary
 /* @var $config phpbb\config\db */
 $config = $phpbb_container->get('config');
-set_config(null, null, null, $config);
```

Comment:
Ok, I will add it back. Same for `set_config_count()` and `request_var()`.
Can I use the same ticket for `set_config()`?